includes/admin/OrderSettings.php

```diff
@@ -251,7 +251,7 @@ private function setAgeCheck(): void
             "{$this->carrier}_" . WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_AGE_CHECK
         );
 
-        $this->ageCheck = $ageCheckOfProduct ?? $ageCheckFromSettings;
+        $this->ageCheck = $ageCheckFromSettings ?? $ageCheckOfProduct;
     }
```

Comment:
Moet het product niet leidend zijn en zodra die niet gezet is dat hij dan naar de settings kijkt?